sceptre-example/config/special-file-asset-cdk-json.yaml

```diff
@@ -0,0 +1,31 @@
+# This template is an example of using the CDK template handler to deploy with a stack from a CDK
```

Comment:
@zaro0508, Having this example here demonstrates how using Sceptre can tie together CDK projects and other resources into an environment. I don't see how this makes things confusing. Are you able to elaborate on why you find this confusing?

Reply:
it's confusing because files for a non-python project are mixed in with files for a python example.  Also python are put in the sceptre standard `templates` folder while non-python files are not.  Those are all fine except i think new users looking at this handler would be confused if they just looked at `sceptre-example` folder for the first time.